lib/engine/game/g_18_uruguay/step/route_rptla.rb

```diff
@@ -55,13 +55,9 @@ def choices
             choices
           end
 
-          def ship_capacity(train)
-            SHIP_CAPACITY[train.name.partition('+')[0]]
```

Comment:
As you're removing this method from this class, the `SHIP_CAPACITY` constant at the start of this file is no longer needed.

Reply:
Removed